src/Controls/src/Core/TitleBar/TitleBar.cs

```diff
@@ -256,6 +256,16 @@ public TitleBar()
 			}
 		}
 
+		internal void Cleanup()
```

Comment:
Can we include a unit test for this one? 

Just inside the Controls.Unit test project?